git_machete/cmd.py

```diff
@@ -424,6 +425,16 @@ def get_git_dir():
     return git_dir
 
 
+def get_abs_git_dir():
```

Comment:
Mostl likely `get_git_dir` in the current form won't be necessary anymore now that we have this function

Reply:
It turned out to be quite useful in a new implementation.

Reply:
Pls reconsider given the recent comments :)